src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimates/UploadBurdenEstimateTests.kt

```diff
@@ -205,4 +216,12 @@ class UploadBurdenEstimateTests : BurdenEstimateTests()
                 "details" to "median"
         ))
     }
+
+    private fun metadataForCreateWithModelRunParameterSet() = json {
+        obj("type" to obj(
+                "type" to "central-averaged",
+                "details" to "median"
+        ),
+                "model_run_parameter_set" to 1)
+    }
```

Comment:
Maybe format as follows? I original misread it and didn't realise what was going on.

```
private fun metadataForCreateWithModelRunParameterSet() = json {
        obj(
                "type" to obj(
                        "type" to "central-averaged",
                        "details" to "median"
                ),
                "model_run_parameter_set" to 1
        )
    }
```

Reply:
done :+1: